spec/controllers/users/reset_passwords_controller_spec.rb

```diff
@@ -127,9 +123,11 @@
           confirmed: true,
         }
 
-        expect(@analytics).to have_received(:track_event).
+        expect(@analytics).to receive(:track_event).
           with(Analytics::PASSWORD_RESET_PASSWORD, analytics_hash)
 
+        put :update, params: { reset_password_form: form_params }
```

Comment:
Was moving this code around related to the Rails upgrade, or a general improvement?